include/ozo/type_traits.h

```diff
@@ -522,7 +522,7 @@ OZO_PG_DEFINE_CUSTOM_TYPE(custom_type, "code.custom_type")
 
 //...
 // Creating ConnectionSource for futher requests to a database
-const auto conn_source = ozo::make_connection_info("...", regiter_types<custom_type>());
```

Comment:
regiter_types == register_types ?

Reply:
Thanks, Michael!

Roma, could you fix this typo, please?

Reply:
Done

Reply:
Thanks a lot!